CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Changelog for NeoFS Node
 - Internal container cache size from 10 to 1000 (#2600)
 - Transaction witness scope no longer uses CustomGroups relying on more strict Rules now (#2619)
 - New optimized SDK version is integrated (#2622)
+- LocodeDB uses from `locodedb` package (#2610)
```

Comment:
We need better explanation here. There are removed commands (for the `Removed` section), there is a change to the DB and we no longer require DB file for IR. All of this must be communicated to our dear users.